### PR TITLE
fix: bundle esm into commonjs for serving

### PR DESCRIPTION
**Pull request by blacha**

### CODE REVIEW

File: scripts/bundle.js [hunk 1]
```diff
@@ -65,14 +65,15 @@ function bundleJs(basePath, cfg, outfile) {
...
-        `--target=${cfg.target || 'es2018'}`,
+        `--target=${cfg.target || 'es2020'}`,
         `--format=${cfg.format || 'cjs'}`,
         ...Object.entries(cfg.env || {}).map(defineEnv),
         ...Object.entries(DefaultEnvVars).map(defineEnv),
         ...(cfg.external || []).map((c) => `--external:${c}`),
         `--outfile=${outfile}`,
         joinPath(basePath, cfg.entry),
     ];
+    console.log(buildCmd);
```
Comment (Wentao-Kuang) [inline scripts/bundle.js:76]: `console.log` forgot to remove

File: packages/lambda-tiler/src/routes/health.ts [hunk 2]
```diff
@@ -87,6 +86,7 @@ export async function Health(req: LambdaHttpRequest): Promise<LambdaHttpResponse
 
         // Get test tile to compare
         const testBuffer = await getTestBuffer(test);
+        test.buf = testBuffer;
```
Comment (Wentao-Kuang) [inline packages/lambda-tiler/src/routes/health.ts:89]: Is this `test.buf` been used by anything?

File: scripts/bundle.js (hunk 1 above)
Comment (blacha) [reply scripts/bundle.js:76]: I kinda like the build command being logged :D

File: packages/lambda-tiler/src/routes/health.ts (hunk 2 above)
Comment (blacha) [reply packages/lambda-tiler/src/routes/health.ts:89]: I believe it is somewhere, I didnt want to break backwards compatibility

**Review by Wentao-Kuang — APPROVED** [review]

**Pull request merged**